src/test/java/com/dabsquared/gitlabjenkins/workflow/UpdateGitLabCommitStatusStepTest.java

```diff
@@ -48,7 +48,9 @@ public void updateGitlabCommitStatus() throws Throwable {
         String pipelineText =
                 IOUtils.toString(
                         getClass().getResourceAsStream("pipeline/updateGitlabCommitStatus.groovy"));
-        rr.then(j -> _updateGitlabCommitStatus(j, port, pipelineText));
+        rr.then(j -> {
+            _updateGitlabCommitStatus(j, port, pipelineText);
+        });
```

Comment:
@basil I applied this change because it was suggested by Netbeans.  I don't understand how the change resolves the compiler's confusion between `RealJenkinsRule.Step`  and `RealJenkinsRule.Step2`, but I'm confident that you can tell if the change is correct.

We've selected this plugin as one of the plugins that we're encouraging others to adopt through our DevOps World workshop.

Reply:
Looks good to me